generate-client.sh

```diff
@@ -1 +1,5 @@
-./vendor/k8s.io/code-generator/generate-groups.sh all github.com/hobbyfarm/gargantua/pkg/client github.com/hobbyfarm/gargantua/pkg/apis "hobbyfarm.io:v1 terraformcontroller.cattle.io:v1"
+./vendor/k8s.io/code-generator/generate-groups.sh all \
```

Comment:
Seems like this should be prepended with `bash` since the vendered file isn't executable.

Reply:
ahh yes bad copy pasta, this is why we code review, i was focused on the vendor too much. ill fix both

Reply:
Running `go mod vendor` resets the file permissions. What I mean is to execute

    bash ./vendor/...

instead of

    ./vendor/...

That way it doesn't matter if the script is executable.